app/models/hardware.rb

```diff
@@ -4,6 +4,7 @@ class Hardware < ApplicationRecord
   belongs_to  :miq_template,  :foreign_key => :vm_or_template_id
   belongs_to  :host
   belongs_to  :computer_system
+  belongs_to  :physical_switch
```

Comment:
do you need a `:foreign_key => :switch_id` here?

Reply:
I've added that [in the `PhysicalSwitch` object](https://github.com/saulotoledo/manageiq/blob/1d8a44f1967c8c5d098e9af128d47c61ab14568f/app/models/physical_switch.rb#L3). Do not should be there? Also, since I am using `:foreign_key`, it will work if I add the `:inverse_of` as requested by the RuboCop?

Reply:
You're right. Thanks! It is fixed now.